contracts/scripts/foundry/InitializeL1ScrollOwner.s.sol

```diff
@@ -0,0 +1,246 @@
+        owner.grantRole(TIMELOCK_1DAY_DELAY_ROLE, L1_1D_TIMELOCK_ADDR);
+        owner.grantRole(TIMELOCK_7DAY_DELAY_ROLE, L1_7D_TIMELOCK_ADDR);
+
+        owner.grantRole(bytes32(0), L1_14D_TIMELOCK_ADDR);
+        owner.revokeRole(bytes32(0), vm.addr(L1_DEPLOYER_PRIVATE_KEY));
+    }
+
+    function configProxyAdmin() internal {
+        bytes4[] memory _selectors;
+
+        // no delay, security council
+        _selectors = new bytes4[](3);
+        _selectors[0] = ProxyAdmin.changeProxyAdmin.selector;
+        _selectors[1] = ProxyAdmin.upgrade.selector;
+        _selectors[2] = ProxyAdmin.upgradeAndCall.selector;
+        owner.updateAccess(L1_PROXY_ADMIN_ADDR, _selectors, SECURITY_COUNCIL_NO_DELAY_ROLE, true);
+    }
+
+    function configScrollChain() internal {
+        bytes4[] memory _selectors;
+
+        // no delay, scroll multisig
+        _selectors = new bytes4[](5);
+        _selectors[0] = ScrollChain.revertBatch.selector;
+        _selectors[1] = ScrollChain.removeSequencer.selector;
+        _selectors[2] = ScrollChain.removeProver.selector;
+        _selectors[3] = ScrollChain.updateMaxNumTxInChunk.selector;
+        _selectors[4] = ScrollChain.setPause.selector;
+        owner.updateAccess(L1_SCROLL_CHAIN_PROXY_ADDR, _selectors, SCROLL_MULTISIG_NO_DELAY_ROLE, true);
+
+        // delay 1 day, scroll multisig
+        _selectors = new bytes4[](2);
+        _selectors[0] = ScrollChain.addSequencer.selector;
+        _selectors[1] = ScrollChain.addProver.selector;
+        owner.updateAccess(L1_SCROLL_CHAIN_PROXY_ADDR, _selectors, TIMELOCK_1DAY_DELAY_ROLE, true);
+    }
+
+    function configL1MessageQueue() internal {
+        bytes4[] memory _selectors;
+
+        // delay 1 day, scroll multisig
+        _selectors = new bytes4[](2);
+        _selectors[0] = L1MessageQueue.updateGasOracle.selector;
+        _selectors[1] = L1MessageQueue.updateMaxGasLimit.selector;
+        owner.updateAccess(L1_MESSAGE_QUEUE_PROXY_ADDR, _selectors, TIMELOCK_1DAY_DELAY_ROLE, true);
+    }
+
+    function configL1ScrollMessenger() internal {
+        bytes4[] memory _selectors;
+
+        // no delay, scroll multisig
+        _selectors = new bytes4[](1);
+        _selectors[0] = ScrollMessengerBase.setPause.selector;
+        owner.updateAccess(L1_SCROLL_MESSENGER_PROXY_ADDR, _selectors, SCROLL_MULTISIG_NO_DELAY_ROLE, true);
+    }
+
+    function configL2GasPriceOracle() internal {
+        bytes4[] memory _selectors;
+
+        // no delay, scroll multisig
+        _selectors = new bytes4[](2);
+        _selectors[0] = L2GasPriceOracle.updateWhitelist.selector;
+        _selectors[1] = L2GasPriceOracle.setIntrinsicParams.selector;
+        owner.updateAccess(L2_GAS_PRICE_ORACLE_PROXY_ADDR, _selectors, SCROLL_MULTISIG_NO_DELAY_ROLE, true);
+    }
+
+    function configMultipleVersionRollupVerifier() internal {
+        bytes4[] memory _selectors;
+
+        // no delay, security council
+        _selectors = new bytes4[](1);
```

Comment:
Doc also says `renounceOwnership` (but I think it's a mistake).

Reply:
fix misktake doc.